utils/build_swift/tests/build_swift/argparse/test_types.py

```diff
@@ -192,13 +192,17 @@ def test_valid_swift_version(self):
         self.assertIsInstance(version, Version)
         self.assertEqual(version.components, (100, 0, 999, 1))
 
+        version = swift_version_type('5.4.0.300.205')
+        self.assertIsInstance(version, Version)
+        self.assertEqual(version.components, (5, 4, 0, 300, 205))
+
     def test_invalid_swift_version(self):
         swift_version_type = types.SwiftVersionType()
 
         with self.assertRaises(ArgumentTypeError):
             swift_version_type('2')
-            swift_version_type('1.8.0.2.3')
-            swift_version_type('100.0.56.1.85')
+            swift_version_type('1.8.0.2.3.1')
```

Comment:
Not familiar with `assertRaises` semantics, but wondering if each of `swift_version_type` invocation should get its own context manager to ensure we get an exception for each and don't stop testing the first case